net/sing-box/files/sing-box.init

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+
+NAME="sing-box"
+PROG="/usr/bin/sing-box"
+
+start_service() {
+	config_load "$NAME"
+
+	local enabled user group conffile workdir
+	config_get_bool enabled "main" "enabled" "0"
+	[ "$enabled" -eq "1" ] || return 0
```

Comment:
```suggestion
	[ "$enabled" -eq "1" ] || return 1
```

Reply:
Service is not enabled, and init refuses to start it, this is not "success".

https://github.com/search?q=repo:openwrt/packages+return+1+language:Shell&type=code&p=1&l=Shell